openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -68,6 +68,12 @@ class AccountView(APIView):
 
                 * goals: null or textual representation of goals
 
+                * bio: null or textural representation of user biographical information ("about me")
+
+            For all text fields, clients rendering the values should take care to HTML escape them to avoid
```

Comment:
@symbolist A couple of things to note and test in your account and profile pages--

1) Whatever text is supplied will be stored as-is. So you should make sure to HTML-escape the text on display (so if the user typed in HTML, what they see will be the literal HTML code, not what they would see if the HTML was rendered as HTML). This is necessary to avoid script injections.

2) We think that unicode characters will update and display properly. Please test some non-ASCII characters and let us know if there is an issue.